templates/eap-xp1-amq-s2i.json

```diff
@@ -74,6 +74,7 @@
             "displayName": "Galleon layers",
             "description": "Comma separated list of Galleon layers to provision a server.",
             "name": "GALLEON_PROVISION_LAYERS",
+            "value": "cloud-profile,remoting",
```

Comment:
@luck3y , the supported layer is 'cloud-server'. If remoting is needed we should add that as a dependency to the layer that needs it.

Reply:
+1 yeah, this was just left in as a reminder on the remoting part, I've tried cloud-server and a lot of other combinations before tracing through the messaging script and finding remoting.